### PR TITLE
Ensure entity name is used in Nats subject name, add telemetry test

### PR DESCRIPTION
**Pull request by twelch**
## 📚 Purpose
This fixes a bug where the entity name passed in was not being used in the Nats subject.  It was still using hardcoded 'event' name.  Let me know if there's a straighforward way this can be tested to avoid.  I added an additional telemetry test which doesn't actually test this code path for the fix.

## 📦 Impacts:
mds-web-sockets

## ✅ PR Checklist

### CODE REVIEW

File: packages/mds-web-sockets/ws-server.ts [hunk 1]
```diff
@@ -105,6 +105,6 @@ export const WebSocketServer = async <T extends readonly string[]>(entityTypes?:
   }
 
   supportedEntities.forEach(async e => {
-    await stream.NatsStreamConsumer(`${TENANT_ID}.event`, processor).initialize()
+    await stream.NatsStreamConsumer(`${TENANT_ID}.${e}`, processor).initialize()
   })
```
Comment (drtyh2o) [inline packages/mds-web-sockets/ws-server.ts:109]: This feels like it should probably be 
```
await Promise.all(
  supportedEntities.map(e => stream.NatsStreamConsumer(`${TENANT_ID}.${e}`, processor).initialize()
)
```

Comment (twelch) [reply packages/mds-web-sockets/ws-server.ts:109]: When I try to switch over to a map call as suggested @drtyh2o I get the following error on it.  I'm not quite grokking.  @avatarneil I know you want to land this today perhaps you have thoughts?  Or I can land without.  I don't have time to dive in on it at the moment.

This expression is not callable.
  Each member of the union type '(<U>(callbackfn: (value: string, index: number, array: readonly string[]) => U, thisArg?: any) => U[]) | (<U>(callbackfn: (value: "event" | "telemetry", index: number, array: readonly ("event" | "telemetry")[]) => U, thisArg?: any) => U[])' has signatures, but none of those signatures are compatible with each other.ts(2349)

Comment (twelch) [reply packages/mds-web-sockets/ws-server.ts:109]: It's interesting that forEach worked and map does not, perhaps forEach is a method that's more broadly supported

Comment (twelch) [reply packages/mds-web-sockets/ws-server.ts:109]: Here's the type signature

async <T extends readonly string[]>(entityTypes?: T)

Comment (twelch) [reply packages/mds-web-sockets/ws-server.ts]: This works...

` 
await Promise.all(
    (supportedEntities as readonly string[]).map(e =>
      stream.NatsStreamConsumer(`${TENANT_ID}.${e}`, processor).initialize()
    )
  )`

**Pull request merged**